### ISSUE TITLE
Advanced Actions Talk - Checklist

### ISSUE THREAD

**Issue by mickeygousset** [devopselvis/github-actions-presentation#23]
## To Do

- [ ] Add build status light to build and release demos - travel light
- [x] Add code scanning to build and release demos
- [x] Add steps for working with GH CLI for build and release demos
- [x] Confirm composite action and reusable workflow work correctly
- [x] Custom Actions - Dad Joke
  - [x] Identify the repos that related to those
  - [x] Add workflow to this repo that uses the dad joke, that others can run
- [ ] Confirm "star the repo" demo works with travel light
- [x] Debugging workflows
  - [x] Adding debugging variables
  - [x] Nektos/ACT

## General

- [ ] Create release once all demos are working, of the repo in the "start" state
- [ ] Make a list of things to check in the settings and actions tabs before starting the demo
  - [ ] Disable appropriate workflows
  - [ ] Disable appropriate branch protection rules
  - [ ] Open tabs to Azure Portal and OIDC config
  - [ ] Delete any existing branches

## Stretch Goals

- [ ] Issue Ops demo to create a new repo
- [ ] Create Issue Ops Automation to configure this repo to start a demo
  - [ ] Basically do all the stuff in our check list
